tests/main/completion/task.yaml

```diff
@@ -1,7 +1,9 @@
 summary: Check different completions
 
-# ppc64el disabled because of https://github.com/snapcore/snapd/issues/2502
-systems: [-ubuntu-core-16-*, -ubuntu-*-ppc64el]
+systems:
```

Comment:
Ok, I remember my intention for this. The comment above only applies for ppc so I moved this to an expanded yaml array to put the comment directly above the ppc line.